ui/src/components/identities/IdentityForm.tsx

```diff
@@ -24,6 +22,34 @@ const initialValues: IdentityFormData = {
   type: IdentityType.BJJ,
 };
 
+function getNetworkFormValues(
+  blockchains: Blockchain[],
+  blockchainName?: string,
+  networkName?: string
+): Partial<IdentityFormData> | null {
+  const foundNetwork = blockchains.find(({ name }) => name === blockchainName);
+  const selectedNetwork = foundNetwork ?? blockchains[0];
+
+  if (!selectedNetwork) {
+    return null;
+  }
+
+  const { name, networks } = selectedNetwork;
+  const network = networks.find(({ name }) => name === networkName) || networks[0];
+  const parsedCredentialStatusType = credentialStatusTypeParser.safeParse(network.rhsMode[0]);
```

Comment:
We don't need to parse this.

`network.rhsMode[0]` is already of type `CredentialStatusType `